pkgs/development/python-modules/django-filter/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv
+, fetchPypi, fetchpatch
+, buildPythonPackage
+, django, python, django-crispy-forms, mock, djangorestframework
+}:
+buildPythonPackage rec {
+
+  pname = "django-filter";
+  version = "2.1.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "3dafb7d2810790498895c22a1f31b2375795910680ac9c1432821cbedb1e176d";
+  };
+
```

Comment:
`ERROR: django-filter requires Python '>=3.4' but the running Python is 2.7.16`

You'll have to add `disabled = pythonOlder "3.4";` to prevent it trying to build this package.

Reply:
Got that one knocked out.